tests/post/steps/test_liveness.py

```diff
@@ -0,0 +1,27 @@
+import pytest
+from pytest_bdd import scenario, given, when, then, parsers
+
+# Scenarios
+@scenario('../features/pods_alive.feature', 'get pods')
+def test_pods(host):
```

Comment:
Where does the `host` fixture come from?

Reply:
host is a fixture from testinfra, it basically gives you a runner on the remote host passed to pytests